app/SignUpSreen.jsx

```
@@ -0,0 +1,285 @@
+                toggleShow={() => setShowPass(!showPass)}
+              />
+              <PasswordInput
+                placeholder="Nhập lại mật khẩu"
+                value={confirmPass}
+                onChangeText={setConfirmPass}
+                show={showConfirmPass}
+                toggleShow={() => setShowConfirmPass(!showConfirmPass)}
+              />
+
+              <Text style={styles.termText}>
+                Bằng cách nhấn "Tạo tài khoản", bạn đồng ý với điều khoản sử dụng.
```

ESLint

[error] `"` can be escaped with `&quot;`, `&ldquo;`, `&#34;`, `&rdquo;`.

(react/no-unescaped-entities)